tools/src/test/resources/1.5/valid-compositions-1.5.textproto

```diff
@@ -39,11 +39,13 @@ dependencies {
   }
 }
 compositions {
+  bom_ref: "composition-1"
   aggregate: AGGREGATE_COMPLETE
   assemblies: "pkg:maven/partner/shaded-library@1.0"
   dependencies: "acme-application-1.0"
 }
 compositions {
+  bom_ref: "composition-1"
```

Comment:
please revert this line.
we need a showcase where no (optional) `bom-ref` property is set

Reply:
Good catch. Updated test cases.